crates/stages/stages/src/test_utils/test_db.rs

```diff
@@ -162,8 +162,14 @@ impl TestStageDB {
             writer.append_header(header.header(), td, &header.hash())?;
         } else {
             tx.put::<tables::CanonicalHeaders>(header.number, header.hash())?;
-            tx.put::<tables::HeaderTerminalDifficulties>(header.number, td.into())?;
+            // Note: HeaderTerminalDifficulties table is read-only in database after
```

Comment:
for future prs,

these docs are totally meaningless because the code doesnt have any references to HeaderTerminalDifficulties so after this is merged this note will just be confusing.